Makefile

```diff
@@ -76,6 +80,9 @@ deploy-ocp:
 	oc apply -f deploy/crd/
 	oc apply -f deploy/examples/job.yaml
 	hack/revert-deploy-files.sh ${OPERATOR_IMAGE} ${KUEUE_IMAGE}
+	@echo "Waiting for Kueue Controller Manager..."
```

Comment:
What is meant by @echo or @timeout?

Reply:
https://www.gnu.org/software/make/manual/make.html#Echoing